doc/faq.txt

```diff
@@ -162,6 +162,12 @@ but requires that all nodes in the graph have a C implementation:
     f = function([x], (x + 1.) * 2, mode=theano.Mode(linker='c'))
     f(10.)
 
+New GPU backend using libgpuarray
+---------------------------------
+
+The new theano GPU backend (:ref:`gpuarray`) uses ``config.gpuarray.preallocate`` for GPU memory allocation. 
+
+Likewise, :ref:`cuda` uses ``config.lib.cnmem`` for GPU memory allocation. 
```

Comment:
The name "cuda" should not be used to differentiate the back-end. The new back-end use device=cuda. That would be confusion. I would replace 
:ref:`cuda`

by "the old back-end"